docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -71,36 +71,70 @@ estimator = Estimator()
-which returns an instance of [`qiskit.providers.JobV1`](../api/qiskit/qiskit.providers.JobV1). You can get the results from the job (as a [`qiskit.primitives.EstimatorResult`](../api/qiskit/qiskit.primitives.EstimatorResult) object)
+which returns an instance of [`qiskit.providers.JobV1`](../api/qiskit/qiskit.providers.JobV1). 
+
+The V2 primitives take one or more primitive unified blocs (PUBs) as the inputs. Each PUB is a tuple that contains a single circuit and data broadcasted to that circuit.
+
+<Tabs>
+  <TabItem value="EstimatorV2" label="EstimatorV2">
+    ```python
+job = estimator.run([(qc, observable)])
+result = job.result()
+print(result)
+```
+
+You can get the results from the job (as a [`qiskit.primitives.EstimatorResult`](../api/qiskit/qiskit.primitives.EstimatorResult) object)
 with the [`qiskit.providers.JobV1.result`](../api/qiskit/qiskit.providers.JobV1#result) method.
 
 ```python
+print(f" > Metadata: {result[0].metadata}")
+```
+  </TabItem>
+  <TabItem value="EstimatorV1" label="Estimator (V1)">
+    ```python
 job = estimator.run(qc, observable)
 result = job.result()
 print(result)
 ```
+You can get the results from the job (as a [`qiskit.primitives.EstimatorResult`](../api/qiskit/qiskit.primitives.EstimatorResult) object)
+with the [`qiskit.providers.JobV1.result`](../api/qiskit/qiskit.providers.JobV1#result) method.
 
 ```python
 EstimatorResult(values=array([4.]), metadata=[{}])
 ```
+  </TabItem>
+</Tabs>
+
+
 
 This example only uses one [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) and one observable. If you want to get expectation values for multiple circuits and observables, you can pass a `list` of [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit)s and a list of `BaseOperator`s to the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method. Both `list`s must have the same length.
 
 #### Get the expected value
 
-From these results you can extract the expected values with the attribute [`qiskit.primitives.EstimatorResult.values`](../api/qiskit/qiskit.primitives.EstimatorResult#values).
+
+<Tabs>
+  <TabItem value="EstimatorV2" label="EstimatorV2">
+    ```python
+print(f" > Expectation value: {result[0].data.evs}")
+print(f" > Metadata: {result[0].metadata}")
+```
+  </TabItem>
+  <TabItem value="EstimatorV1" label="Estimator (V1)">
+  From these results you can extract the expected values with the attribute [`qiskit.primitives.EstimatorResult.values`](../api/qiskit/qiskit.primitives.EstimatorResult#values).
 
 [`qiskit.primitives.EstimatorResult.values`](../api/qiskit/qiskit.primitives.EstimatorResult#values) returns a `numpy.ndarray`
 whose `i`<sup>th</sup> element is the expectation value corresponding to the `i`<sup>th</sup> circuit and `i`<sup>th</sup> observable.
```

Comment:
In this case, we are instead getting the 0th item from a PrimitiveResult, which is a PubResult, and then looking at the evs field PubResult.data, which is itself an ndarray.

Reply:
That's happening in V1 or V2?

Reply:
> In this case, we are instead getting the 0th item from a PrimitiveResult, which is a PubResult, and then looking at the evs field PubResult.data, which is itself an ndarray.

Is your comment about V1 or V2?

Reply:
Addressed in [f21db70](https://github.com/Qiskit/documentation/pull/812/commits/f21db70cc23e1f49a385481afb2c0b002f812294).